### ISSUE TITLE
Remember my password

### ISSUE THREAD

**Issue by TeSantana** [Common-Voice/sentence-collector#213]
Hi,
I was trying to login into https://common-voice.github.io/sentence-collector/#/login 
![imagen](https://user-images.githubusercontent.com/50135707/57234783-b1290d00-7021-11e9-9446-186fc5032e8d.png)

I forgot my password. I've tried with different ones until I logged in finally. 
I was looking for some link to help me to find a way to get a new one, but it doesn't exist. 

Nice to have a way to generate a new password, to avoid to sing up again with a different account - and miss all your number of sentences added and reviewed. 

Ta!

**Comment by MichaelKohler** [comment]
Our goal is still to eventually implement Auth0, so this won't be necessary anymore once #208 is done. Might take a while to get there though..
